{{cookiecutter.project_slug}}/README.md

```diff
@@ -1,3 +1,109 @@
 # {{ cookiecutter.project_name }}
 
 [![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
+[![Actions Status][actions-badge]][actions-link]
+[![Documentation Status][rtd-badge]][rtd-link]
+
+[![PyPI version][pypi-version]][pypi-link]
+[![Conda-Forge][conda-badge]][conda-link]
+[![PyPI platforms][pypi-platforms]][pypi-link]
+
+[![GitHub Discussion][github-discussions-badge]][github-discussions-link]
+[![Gitter][gitter-badge]][gitter-link]
+
+<!-- prettier-ignore-start -->
+[actions-badge]:            https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.project_slug}}/workflows/CI/badge.svg
+[actions-link]:             https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.project_slug}}/actions
+[conda-badge]:              https://img.shields.io/conda/vn/conda-forge/{{cookiecutter.project_slug}}
+[conda-link]:               https://github.com/conda-forge/{{cookiecutter.project_slug}}-feedstock
+[github-discussions-badge]: https://img.shields.io/static/v1?label=Discussions&message=Ask&color=blue&logo=github
+[github-discussions-link]:  https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.project_slug}}/discussions
+[gitter-badge]:             https://badges.gitter.im/{{cookiecutter.project_slug}}/community.svg
+[gitter-link]:              https://gitter.im/{{cookiecutter.project_slug}}/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge
+[pypi-link]:                https://pypi.org/project/{{cookiecutter.project_slug}}/
+[pypi-platforms]:           https://img.shields.io/pypi/pyversions/{{cookiecutter.project_slug}}
+[pypi-version]:             https://img.shields.io/pypi/v/{{cookiecutter.project_slug}}
+[rtd-badge]:                https://readthedocs.org/projects/{{cookiecutter.project_name}}/badge/?version=latest
+[rtd-link]:                 https://{{cookiecutter.project_slug}}.readthedocs.io/en/latest/?badge=latest
```

Comment:
A question: should we keep all of this? We don't really recommend RTD over github pages so feels like we shouldn't make a choice here, but leave it to the user.

But then our cookie-cutter becomes very complicated: should we have "are you using GH discussions or slack?" and "are you using github pages or RTD?" type questions? Or just leave a lot of this commented out for the user to uncomment further down the line.

Reply:
It would definitely be nicer to offer the option of github pabes vs rtd, but probably in a separate pr. Maybe remove the links for now?

And the PyPI badges could probably just be commented out and a comment added that says to uncomment once the package has been published

Reply:
Yeah agreed. Let's put that onto 
* #16